src/viur/core/bones/string.py

```diff
@@ -229,6 +241,29 @@ def buildDBSort(
                 dbFilter.order(order)
         return dbFilter
 
+    def natural_sorting(self, value: str | None) -> str | None:
+        """Implements a default natural sorting transformer.
+
+        The sorting is according to DIN 5007 Variant 2
+        and sets ö and oe, etc. equal.
+        """
+        if value is None:
+            return None
+        assert isinstance(value, str)
+        if not self.caseSensitive:
+            value = value.lower()
+
+        # DIN 5007 Variant 2
+        return value.translate(str.maketrans({
```

Comment:
This is useful! [translate](https://docs.python.org/3/library/stdtypes.html#str.translate) should be used on several ugly and stupid replace-cascades.